package/toltec-bootstrap/toltecctl

```diff
@@ -169,48 +313,72 @@ uninstall-force() {
     reboot
 }
 
-uninstall() {
-    # Remove all installed packages after prompting user
-    if [[ $# -gt 0 && ($1 = "-f" || $1 = "--force") ]]; then
-        uninstall-force
-        return
-    fi
-
-    read -p "This will wipe out all files in '$toltec_dest'. Continue? [y/n] " -n 1 -r
-    echo
-
-    if [[ $REPLY =~ ^[Yy]$ ]]; then
-        uninstall-force
-        return
-    fi
-
-    echo "Canceled"
-}
-
 help() {
     echo "Usage: $(basename "$0") COMMAND
 Manage your Toltec install. Available commands:
 
-    reenable        Re-enable Toltec after a system update.
-    set-path        Fix PATH value if needed.
-    uninstall       Permanently remove Toltec."
+    generate-opkg-conf      Rebuild the Opkg configuration file.
+    switch-branch [BRANCH]  Change the current branch to BRANCH.
+    reenable                Re-enable Toltec after a system update.
+    uninstall               Permanently remove Toltec."
 }
 
-if [[ $# -eq 0 ]]; then
-    help
-    exit 1
-fi
-
-action="$1"
-shift
-
-case $action in
-    reenable) reenable "$@" ;;
-    set-path) set-path "$@" ;;
-    uninstall) uninstall "$@" ;;
-    *)
-        echo -e "Error: Invalid command '$action'\n"
+if [[ $0 = "${BASH_SOURCE[0]}" ]]; then
+    if [[ $# -eq 0 ]]; then
         help
         exit 1
-        ;;
-esac
+    fi
+
+    action="$1"
+    shift
+
+    case $action in
+        generate-opkg-conf)
+            generate-opkg-conf && opkg update
+            ;;
+
+        switch-branch)
+            if [[ ($# -eq 1 && $1 = "testing") || ($# -eq 2 && $1 = "testing" && $2 != "-f" && $2 != "--force") ]]; then
+                cat << MSG
+Using packages from the testing branch may cause breakage or data loss.
+Please make sure you read the following before switching to testing:
+
+    <https://github.com/toltec-dev/toltec/blob/stable/docs/branches.md>
+
+MSG
+                read -p "Continue? [y/n] " -n 1 -r
+                echo
+
+                if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+                    echo "Canceled"
+                    exit 1
+                fi
+            fi
+            switch-branch "${1:-stable}"
+            generate-opkg-conf && opkg update
+            ;;
+
+        reenable)
+            reenable
+            ;;
+
+        uninstall)
+            if [[ $# -eq 0 || ($1 != "-f" && $1 != "--force") ]]; then
+                read -p "This will wipe out all files in '$toltec_dest'. Continue? [y/n] " -n 1 -r
```

Comment:
```suggestion
                read -p "This will wipe out all files in '$toltec_dest'. Continue? [y/N] " -n 1 -r
```

The implicit selection of "No" should be communicated with an upper case letter.